examples/seismic/tutorials/10_nmo_correction.ipynb

```diff
@@ -0,0 +1,504 @@
VqB/mva3Un20ViWji0dAgg/gUy3jt7F4PTHXOlY4JUS5EHn+DbG1t/fodn2974usfrb/r/W/9b3Wlv/zDZV93rtOU3kU/poSLxJ4iIF8/hkbHVr38wQ9ePasgdKu+B/oTh9fOab3z+PD7Sj8DHbp4qLwZo3aJ+eOzVT1JUAix48U9Kxau5ZYhSj+Hvi0c/ZDRtar0Y2vnoCfubdXdrT3No7JSqDh12LmDBy6fm9++sGL3AtVTnXM1jYd+5NE9Uor/7qO13/izXz/+T7P7H5/92Ff/ori4uL0921UX9CMFMi8LiIBk/eiKXnzwuZNqbkuo9MGvPRfPeOCeHzEhavTnLHWr+ojG+48Zl5jfiMeU1GH4wsf1yzduieyKXx1TUgEK9T0B9ENGF8vXjzPnryx7/pRXniRYUqI9y9x7f8aJxTt7fuq9qllrZNznUc29VuKnj2z6u3//asWMDw68IoSorq5uaGiY3i2z0A9rp7JdWARk6sfmfQOrXv5AwaKH/gRThQ9+fef8e3Na71R9z4+oLh4qV+kvHdzS//js+OkjqsbIye6e2qqGtw+qucZdVaspVzIB9EMGcMn6sbVz8MHnTiq4K3wKy9ZW7WyrujolN2pMqUvKS+Ohuave+r6XnumRUkfjYo9a/T6P2qPNFf4d2rlpcE3Z2W3fC4VCM2bMMB4qMtXr1NEPhT1I0VkSkKMf8bEbDS2nlV2kV12tXZun6O/g4OE5rXeqvr2VccKVmt96DPAj21acj8xXteghhHhzz5G6pc9eGlWz6qLo00exCgigHzKgS9OP0SvXwlui6hc9hoa0U63Kyjx4b6uei2cK4WwrIYT6M4+tY+NGPPbRi9/++aq/adCv3uno6AiFQrNmzTLS9Lk4rQ79sPJku7AIyNGPH7x6tv1dNQ+z0yJ2Y6OqTjFuOaj6x6AOfd1DEX8d/fDLD4+0rFLVC0KItw8eW7NyEydcKeyC4BSNfsjoazn6MTgSX/b8qXd7RmU0yaEMY9EjEnFIouStK9eurHrr+7dum+P5s62M263MFkLZmceZOujcM3NjR9uEEN3d3ZWVlbW1tfoTXJpDoVDNZL+boh+ZqLLf+wQk6MfWzsHN+wbUoKit1Zap1f3d3jbvjf5OdeVrIU25e4y01Y9sW6EQwrn+C7VVDbiHwi4IVNHoh4zulqAfgyPx8JbomfNXZLQnUxmxmLZ2X1bmwUvMm6M7PPDQ3EzgrPu9tehhrZkQIn76SP/js1N2hsPhUChkrIpEIpHa2lrbh4egHynceFlABCTox/ynuxU8UlA7xzMmZs5Ue4rsb/3rH6r+MFQIoeyRgkKI65eHe5d9WuE5V0KIzetadjXvV90RlB8UAuiHjJ7Ot37Ex24sjpwYHInLaEymMoaGRGmpB++re3z4/YrdC6r2flv1WcWZwJn7h4SoVHu7FbMqmTbGzn/QF745/d2+vj7jevSSkpJQKGQ8Rr2jo8N6kTr6kc6NPYVCQIJ+/MOao/GxGwqARKOiqEhBuZYib902R/W9B0uEUHnJh+0vOxZCMjY3rn3l9Ra1a1AymkkZHiGAfsjoiHzrR/u7Qz949ayMlmQqo69Pu1+Kx25vZT5J8LWzezNV3DP7G4UoEsJzZ6yl8NFuB/lYScpO68uOjo7q6mrj2vRZs2bNmDHDfHwh+mEFxXZhEZCgH09s6+k8MaIGS1GR2uv01r23cfnPv6em7eOlVqoNvzfisbMPfUrt6sehA0fXrNyktBcoPEAE0A8ZnZ1v/XjxwMCLBxSdNCyEMNY9ulT+dJTei2/0d3r+SYJGrY3rHauFUHnJYzpA2z3DLz/88etrbd9K2dnX11dWVlak/6oai8WKi4u/9vc1PL4qhRIvC4WABP1QeQJta6soL1d7/tUTh9cqXaPu0u8xqDIIj7SsUnvduRBiw5MvsQBSKEGp0OuJfsjowXzrx7s9o+EtURktsS2jpsZT51wNXx1Z9Ebt7W3zVK/m28Ky7hwSolo/28qLD0WxVtTYvnxk+0fr70rf77DHWAaJRCKhUOjeecvQDwdWvOVlAhL0QwgxOBJfHDkRae9TcBGIccuQ5maFvWBcobf8599TFLq7hChWuwYy0rLqo/V3jZ3/QGEvbFz7ysoljSe7exTWgaKDQECafpx46paQ/V/VTt+Dzrd+CCFU3rFR9cK99fPzRn/nrdvmNJ3cat3pye0G/WwrZXe6nCqTi/s2nHtm7rTPDejo6FhX/+NDbxydarmkh4AXCMjRD6OlOw59NP/p7oaW0/u7h6W2PRrV1kCKi7U78A6pWQe4cu3Kuvc23rptzq3b5qx7b6P0C/b69AvwioVoFELN05YuH9neF775fGT+5SPbpfa+pbCT3T0rlzTWVjXsat4/fOFjyztsQiBnBKTph6XGO6tCoVueOmHZ4/dNCfphGMgT23oUXIBeWira25X3oXFf3dvb5kmfsaba9Hb9N7aagjjbSghxIx4bfvnhoRcWTbWdKemNJ+kuurtuzcpNu5r3f/h+b0oCXkLAswRk6ocBYX/3cEPL6flPd6/f3Sv1durd3aK6WrsXVkWFULcYcnz4/eU//95nfnzr7W3zNh5rkhvVu/V16VlClKu6Hdalg1s+Wn9X77JPD72wKN6v5unjvT2Dm9e1LLq7rn75xn1th3gQoWejU4FWDP2Q0XFy9EMI0Xli5L4NxxtaTndFL8pomFFGY6M2XSn9G746Mr994Zp31imtxaSF9+nzWakHH+iRqepj5z8YXFPm8nqPTJlY9w9f+PjQgaOb17WsWLh2wdxH65dvbN60m5OyrIjY9iAB+fphQBi9cm3X4fPhLdEFP/zF+t29vzh7WR6cpiZtMWTGDO3/TcpuSvtGf+dDb4Zv3jL7q+0L5bV9vKRmISqEmKH/v0kI2Q8CvzbSf3Hfhv7HSvrCNw+//HD89BHpBLQC3z54bMOTL917x8o1Kze17+TWWEo6wYeFekI/9BOzLOdgpSyPpLytvWv+WY5K652
```

Comment:
Receivers are basically free to model so you can do `nrcv=251` or something to have the classical hyperbolic even show up nicely in the plot.